app/src/androidTest/java/org/mozilla/fenix/ui/PwaTest.kt

```diff
@@ -23,6 +25,7 @@ class PwaTest {
     private val emailLink = "mailto://example@example.com"
     private val phoneLink = "tel://1234567890"
     private val shortcutTitle = "TEST_APP"
+    private val pwaTask = "ComponentInfo{org.mozilla.fenix.debug/org.mozilla.fenix.customtabs.ExternalAppBrowserActivity}"
```

Comment:
The package - `org.mozilla.fenix.debug` will be different when running these tests for different flavors.
The package provides no useful information so it can be elided from the check.
To know if the current task of the app is for a custom tab / PWA just checking the class name should be enough, with something like
```kotlin
activityManager.appTasks[0].taskInfo.topActivity!!.className == ExternalAppBrowserActivity::class.java.name
```

Reply:
Thanks for the heads-up !